schema.xsd

```diff
@@ -166,20 +168,52 @@
         <xs:attribute name="relevance" type="xs:decimal" use="required"/>
     </xs:complexType>
 
+    <!-- Range slider filters. Filters "min"/"max" may not have any values if there is nothing to filter for. -->
```

Comment:
'... any values if there is nothing to filter for.' -> '... any values if there is nothing to filter'